test/integration/sanity/proxy-http.test.js

```diff
@@ -0,0 +1,44 @@
+describe('sanity test', function () {
```

Comment:
Great. Now we need to go to the next step.

How do we verify whether our proxy was actually used?